test/fun_with_flags/ui/router_test.exs

```diff
@@ -198,6 +198,39 @@ defmodule FunWithFlags.UI.RouterTest do
     end
   end
 
+  describe "CSP nonce assign key option" do
+    test "if csp_nonce_assign_key is set, the CSP nonce is rendered in the script and link tags" do
+      {:ok, true} = FunWithFlags.enable :coconut
+
+      csp_opts = [csp_nonce_assign_key: :pineapple]
+
+      conn =
+        conn(:get, "/flags/coconut")
+        |> Plug.Conn.assign(:pineapple, "mango")
```

Comment:
Let's add a comment to explain what's happening here.

Why this can be confusing: manipulating `conn` here is basically preparing the _request_ data, not the response, because this happens before calling `Router.call(...)`.

Since this library's "server backend" is self-contained in its Router, I guess that here you expect the nonce `"mango"` to be generated and set in the assigned by a different `plug`, executed before this FWF router, in the host-application's router.

That's ok, but let's make it clear with a comment.